src/lib/chat/matrix-client.ts

```diff
@@ -119,6 +119,10 @@ export class MatrixClient implements IChatClient {
     return dmConversationIds.includes(room.roomId) || !!room.getDMInviter();
   };
 
+  private isDeleted(m) {
```

Comment:
Probably should use a better variable name here. What is "m"?

Reply:
Good shout! Updated to `event`